src/egon/data/datasets/industrial_gas_demand.py

```diff
@@ -460,104 +472,165 @@ def insert_industrial_gas_demand_egon100RE():
-        # to save processing cost
-        H2_total_PES *= 0.01855683050330346
-        CH4_total_PES *= 0.01855683050330346
-
-    H2_total = industrial_gas_demand_H2["p_set"].apply(sum).astype(float).sum()
-
-    industrial_gas_demand_CH4["p_set"] = industrial_gas_demand_H2[
-        "p_set"
-    ].apply(lambda x: [val / H2_total * CH4_total_PES for val in x])
-    industrial_gas_demand_H2["p_set"] = industrial_gas_demand_H2[
-        "p_set"
-    ].apply(lambda x: [val / H2_total * H2_total_PES for val in x])
-
-    # consistency check
-    total_CH4_distributed = sum(
-        [sum(x) for x in industrial_gas_demand_CH4["p_set"].to_list()]
-    )
-    total_H2_distributed = sum(
-        [sum(x) for x in industrial_gas_demand_H2["p_set"].to_list()]
-    )
+    new_id = db.next_etrago_id("load")
+    loads_to_db["load_id"] = range(new_id, new_id + len(loads_to_db))
 
-    print(
-        f"Total amount of industrial H2 demand distributed is "
-        f"{total_H2_distributed} MWh. Total amount of industrial CH4 demand "
-        f"distributed is {total_CH4_distributed} MWh."
-    )
-    msg = (
-        f"Total amount of industrial H2 demand from P-E-S is equal to "
-        f"{H2_total_PES}, which should be identical to the distributed amount "
-        f"of {total_H2_distributed}, but it is not."
+    # Deal with egon_etrago_load_timeseries
+    load_gas_t = pd.read_csv(
+        "data_bundle_powerd_data/egon_etrago_load_t_gas.csv",
     )
-    assert round(H2_total_PES) == round(total_H2_distributed), msg
+    load_gas_t = load_gas_t[load_gas_t["scn_name"] == scn_name]
+    load_gas_t["load_id"] = load_gas_t["load_id"].map(loads_to_db["load_id"])
 
-    msg = (
-        f"Total amount of industrial CH4 demand from P-E-S is equal to "
-        f"{CH4_total_PES}, which should be identical to the distributed amount "
-        f"of {total_CH4_distributed}, but it is not."
-    )
-    assert round(CH4_total_PES) == round(total_CH4_distributed), msg
+    # Drop unnecessary columns
+    loads_to_db.drop(columns=["geom", "bus_id"], inplace=True, errors="ignore")
 
-    industrial_gas_demand = pd.concat(
-        [
-            industrial_gas_demand_CH4,
-            industrial_gas_demand_H2,
-        ]
+    # Write results into db
+    engine = db.engine()
+    loads_to_db.to_sql(
+        "egon_etrago_load",
+        engine,
+        schema="grid",
+        index=False,
+        if_exists="append",
     )
-    industrial_gas_demand = (
-        industrial_gas_demand.groupby(["bus", "carrier"])["p_set"]
-        .apply(lambda x: [sum(y) for y in zip(*x)])
-        .reset_index(drop=False)
+
+    load_gas_t.to_sql(
+        "egon_etrago_load_timeseries",
+        engine,
+        schema="grid",
+        index=False,
+        if_exists="append",
     )
 
-    industrial_gas_demand = insert_new_entries(industrial_gas_demand, scn_name)
-    insert_industrial_gas_demand_time_series(industrial_gas_demand)
+    # # read demands

```

Comment:
To be honest, I do not like commented lines that much. It is always not that easy to know why there are commented. Could you maybe think of another option?